### ISSUE TITLE
[Read | LearningMaterials] Enhance CourseProgress read model with handling TaskWasCompleted

### ISSUE THREAD

**Issue by nowakprojects** [CodersCrew/coderscamp#216]
**Dependencies (blockers)**
- [x] Make sure that #213 is done (must be merged before)
- [x] CourseProgress is already done

**Subtasks**
- [ ] Handle in `CourseProgressReadModule` event TaskWasCompleted
- [ ] Update completedCount (increase by one) CourseProgress entity on TaskWasCompleted

https://miro.com/app/board/o9J_lQvnN28=/?moveToWidget=3074457363062824891&cot=14
